src/test/java/com/teamsfinder/userwriteservice/user/service/UserServiceImpTest.java

```diff
@@ -0,0 +1,128 @@
+package com.teamsfinder.userwriteservice.user.service;
+
+import com.teamsfinder.userwriteservice.user.dto.EditUserDto;
+import com.teamsfinder.userwriteservice.user.dto.UserResponseDto;
+import com.teamsfinder.userwriteservice.user.exception.KeyCloakException;
+import com.teamsfinder.userwriteservice.user.exception.UserNotFoundException;
+import com.teamsfinder.userwriteservice.user.keycloak.KeyCloakService;
+import com.teamsfinder.userwriteservice.user.model.AccountType;
+import com.teamsfinder.userwriteservice.user.model.User;
+import com.teamsfinder.userwriteservice.user.repository.UserRepository;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.ArrayList;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+@ExtendWith(MockitoExtension.class)
+class UserServiceImpTest {
+    private static final String USER_KEYCLOAK_ID = "KEYCLOAKID";
+    private static final String EDIT_STRING = "EDITED";
+    private static final String USER_GITHUB = "GITHUB";
+    private static final String USER_PICTURE = "PICTURE";
+
+    @Mock
+    private UserRepository userRepository;
+    @Mock
+    private KeyCloakService keyCloakService;
+
+    @InjectMocks
+    private UserServiceImp userService;
```

Comment:
imo dobra praktyka nazywać to underTest, widać od razu co testujesz bez przyglądania się specjalnego